File: 0888-fair-candy-swap/0888-fair-candy-swap.java
class Solution {
    public int[] fairCandySwap(int[] aliceSizes, int[] bobSizes) {
        
        int aliceSum = sum(aliceSizes);
        int bobSum = sum(bobSizes);

        for(int i = 0 ; i < aliceSizes.length ; i++)
        {
            for(int j = 0 ; j < bobSizes.length ; j++)
            {
                int aliceNewSum = aliceSum - aliceSizes[i] + bobSizes[j];
                int bobNewSum = bobSum - bobSizes[j] + aliceSizes[i];

                if(aliceNewSum == bobNewSum)    
                    return new int[]{aliceSizes[i], bobSizes[j]};
            }
        }

        return new int[]{-1,-1};

    }

    public int sum(int []nums)
    {
        int sum = 0;

        for(int num : nums)
            sum += num;
        
        return sum;
    }
}

File: 0888-fair-candy-swap/0888-fair-candy-swap.js
/**
 * @param {number[]} aliceSizes
 * @param {number[]} bobSizes
 * @return {number[]}
 */
var fairCandySwap = function(aliceSizes, bobSizes) {
    
    let aliceSum = aliceSizes.reduce((acc,el) => acc+=el,0);
    let bobSum = bobSizes.reduce((acc,el) => acc+=el,0);

    for(let i of aliceSizes)
    {
        for(let j of bobSizes)
        {
            let aliceNewSum = aliceSum - i + j;
            let bobNewSum = bobSum - j + i;

            if(aliceNewSum == bobNewSum)
                return [i,j];
        }
    }

    return [-1,-1];
};